cardano-cli/src/Cardano/CLI/EraBased/Options/StakeAddress.hs

```diff
@@ -87,27 +87,18 @@ pStakeAddressRegistrationCertificateCmd :: ()
 pStakeAddressRegistrationCertificateCmd era = do
   forEraInEonMaybe era $ \sbe ->
     caseShelleyToBabbageOrConwayEraOnwards
-      (\shelleyToBabbage -> subParser "registration-certificate"
-            $ Opt.info
-                ( StakeAddressRegistrationCertificateCmd (shelleyToBabbageEraToShelleyBasedEra shelleyToBabbage)
-                    <$> pStakeIdentifier
-                    <*> optional pKeyRegistDeposit
```

Comment:
Why not `return Nothing` for Shelley to Babbage??

Reply:
Hum yeah really smart, I hadn't thought of lifting over the value type rather than the `Parser` type :+1: 

Done the change.